.github/workflows/code-lint-and-prettify.yml

```
@@ -2,52 +1,70 @@
+      pull-requests: write
 
+    steps:
       - name: Checkout code
         uses: actions/checkout@v4
         with:
-          token: ${{ steps.generate_token.outputs.token }}
           fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}  # default; safe to omit
+
+      - name: Import GPG key
+        if: github.event_name == 'push' && secrets.GPG_PRIVATE_KEY != ''
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)